tests/testdata/serve/parallel.ts

```diff
@@ -1,4 +1,4 @@
-console.log("starting serve");
+console.error("starting serve");
 export default {
   fetch(_req: Request) {
     console.log("serving request");
```

Comment:
```suggestion
    console.error("serving request");
```

Should update this too, the test relies on this output, and this PR changed the test helper to read its output from stderr instead of stdout